docs/_static/spec/server.yml

```diff
@@ -655,6 +655,77 @@ paths:
         400:
           $ref: '#/components/responses/400Training'
 
+  /intentEvaluation:
+    post:
+      security:
+      - TokenAuth: []
+      - JWT: []
+      tags:
+      - Model
+      summary: Perform an intent evaluation on a Rasa Stack model
+      description: >-
+        Evaluates NLU intents on a Rasa Stack model. A .tar.gz file has to be
+        provided which must contain the NLU evaluation data and a Rasa Stack
+        model. Example contents of the zipped directory (the NLU evaluation
+        data is contained in data.json):
+
+        evaluation.tar.gz
```

Comment:
I am not convinced this is the best structure, maybe multi-part-form submission is better?

I actually think we shouldn't have these endpoints in the first place and rather use some sort of task management / worker system to submit trainings / evaluations. but I guess for the moment this is the easiest solution.